### DISCUSSION TITLE
Does Lem have log files?  I need to troubleshoot an issue.

### DISCUSSION THREAD

**Discussion by aykaramba** [lem-project/lem#1839] [Q&A]
Lem has recently started hanging when loading CLOG and many many other packages even if I clear ~/.cache/common lisp.

At first I had to make sure that the cache was cleared on every restart because Lem would often hang on loading / compiling various packages, now it just hangs regardless and I have no insight as to why.

Does Lem have logging capability?  If yes, how do I use it?

Thanks.

**Comment by cxxxr** [comment]
You can output logs to a log file by specifying log-filename as an argument when starting lem.
```
lem --log-filename lem.log
```

Also, I think it's better to start a separate process than to use CLOG etc. using the same process as Lem.
Please use the `M-x slime` command.
